week01_intro/homework.ipynb

```diff
@@ -799,24 +799,30 @@
    "source": [
     "from sklearn.linear_model import LogisticRegression\n",
     "lr = LogisticRegression(solver=\"lbfgs\")\n",
-    "#<your code>\n"
+    "#<your code>"
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "pycharm": {
-     "name": "#%%\n"
-    }
-   },
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
-  "hse_dl_year": "2021-fall"
+  "hse_dl_year": "2021-fall",
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.7"
+  }
  },
  "nbformat": 4,
  "nbformat_minor": 2
```

Comment:
Overall, it's not a good idea to push unrelated changes along with the core matter of the PR. I'd appreciate if you could reduce the diff by reverting its redundant parts